emailtemplateservice/tool/src/main/webapp/tools/sakai.emailtemplateservice.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<registration>
+    <tool id="sakai.emailtemplateservice"
+        title="Email template administration"
+        description="emailtemplateservice created by Sakai App Builder Plugin">
```

Comment:
I know that this tool had this description before, but maybe this should be changed? @mpellicer